Conceptpower+Spring/src/main/resources/user.properties

```diff
@@ -0,0 +1 @@
+admin=admin1234,ROLE_CP_ADMIN,enabled
```

Comment:
there needs to be at least one user coming from the pom.xml, which means the two maven variables (admin.user and admin.pw) need to be added.